examples/docker_aws_lambda_example/producer_example.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+/**
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"context"
+	"fmt"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"os"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+var p *kafka.Producer
+
+func main() {
+	// Use signal to kill the client instance.
+	// Adding extensions built is needed to kill in this way:
+	// Reference: https://github.com/aws/aws-lambda-go/issues/318#issuecomment-1019318919
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGTERM, syscall.SIGINT)
+	go func() {
+		fmt.Printf("waiting for the SIGTERM ")
+		s := <-sigs
+		fmt.Printf("received signal %s", s)
+		if p != nil {
+			p.Close()
+		}
+		fmt.Printf("done")
+	}()
+
+	lambda.Start(HandleRequest)
+}
+
+// HandleRequest handles creating producer and
```

Comment:
i think the comments on these two lines are not useful

Reply:
The CI will complain without this comments:

```
examples/docker_aws_lambda_example/producer_example.go:53:1: exported function HandleRequest should have comment or be unexported
Found 1 lint suggestions; failing.
The command "if [[ -f .do_lint ]]; then golint -set_exit_status ./examples/... ./kafka/... ./kafkatest/... ./soaktest/... ; fi" exited with 1.
```